<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="1" total="12" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="13677" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:48Z">
    <groups>
      <group name="smoke">
        <method signature="TestA.firstTestCaseA()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="firstTestCaseA" class="com.bility.test.sanityTests.TestA"/>
        <method signature="YahooTest.yahooSearch()[pri:0, instance:com.bility.test.regressionTests.YahooTest@23ceabc1]" name="yahooSearch" class="com.bility.test.regressionTests.YahooTest"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="TestA.secondWebSiteA()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="secondWebSiteA" class="com.bility.test.sanityTests.TestA"/>
        <method signature="TestA.testYahooSearch()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="testYahooSearch" class="com.bility.test.sanityTests.TestA"/>
        <method signature="TestA.thirdTestCaseA()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="thirdTestCaseA" class="com.bility.test.sanityTests.TestA"/>
        <method signature="TestB.fifthTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="fifthTestCaseB" class="com.bility.test.sanityTests.TestB"/>
        <method signature="TestB.firstTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="firstTestCaseB" class="com.bility.test.sanityTests.TestB"/>
        <method signature="TestB.fourthTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="fourthTestCaseB" class="com.bility.test.sanityTests.TestB"/>
        <method signature="TestB.secondTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="secondTestCaseB" class="com.bility.test.sanityTests.TestB"/>
        <method signature="TestB.thirdTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="thirdTestCaseB" class="com.bility.test.sanityTests.TestB"/>
        <method signature="TestC.firstTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestC@6cc4c815]" name="firstTestCaseB" class="com.bility.test.sanityTests.TestC"/>
        <method signature="TestC.secondTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestC@6cc4c815]" name="secondTestCaseB" class="com.bility.test.sanityTests.TestC"/>
      </group> <!-- sanity -->
    </groups>
    <test name="Command line test" duration-ms="13677" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:48Z">
      <class name="com.bility.test.regressionTests.YahooTest">
        <test-method status="FAIL" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.bility.test.regressionTests.YahooTest@23ceabc1]" name="setUp" is-config="true" duration-ms="0" started-at="2016-02-04T16:03:48Z" finished-at="2016-02-04T16:03:48Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'browserType' is required by @Configuration on method setUp but has not been marked @Optional or defined
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'browserType' is required by @Configuration on method setUp but has not been marked @Optional or defined

	at org.testng.internal.Parameters.createParameters(Parameters.java:154)
	at org.testng.internal.Parameters.createParameters(Parameters.java:361)
	at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:84)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:197)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="yahooSearch()[pri:0, instance:com.bility.test.regressionTests.YahooTest@23ceabc1]" name="yahooSearch" duration-ms="1" started-at="2016-02-04T16:03:48Z" finished-at="2016-02-04T16:03:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- yahooSearch -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.bility.test.regressionTests.YahooTest@23ceabc1]" name="tearDown" is-config="true" duration-ms="0" started-at="2016-02-04T16:03:48Z" finished-at="2016-02-04T16:03:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.bility.test.regressionTests.YahooTest -->
      <class name="com.bility.test.sanityTests.TestA">
        <test-method status="FAIL" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="setUp" is-config="true" duration-ms="0" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:35Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'browserType' is required by @Configuration on method setUp but has not been marked @Optional or defined
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'browserType' is required by @Configuration on method setUp but has not been marked @Optional or defined

	at org.testng.internal.Parameters.createParameters(Parameters.java:154)
	at org.testng.internal.Parameters.createParameters(Parameters.java:361)
	at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:84)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:197)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="firstTestCaseA()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="firstTestCaseA" duration-ms="9" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestCaseA -->
        <test-method status="SKIP" signature="secondWebSiteA()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="secondWebSiteA" duration-ms="1" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- secondWebSiteA -->
        <test-method status="SKIP" signature="testYahooSearch()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="testYahooSearch" duration-ms="1" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testYahooSearch -->
        <test-method status="SKIP" signature="thirdTestCaseA()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="thirdTestCaseA" duration-ms="0" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thirdTestCaseA -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.bility.test.sanityTests.TestA@5d5eef3d]" name="tearDown" is-config="true" duration-ms="3" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.bility.test.sanityTests.TestA -->
      <class name="com.bility.test.sanityTests.TestB">
        <test-method status="PASS" signature="fifthTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="fifthTestCaseB" duration-ms="2005" started-at="2016-02-04T16:03:35Z" finished-at="2016-02-04T16:03:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fifthTestCaseB -->
        <test-method status="PASS" signature="firstTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="firstTestCaseB" duration-ms="2003" started-at="2016-02-04T16:03:37Z" finished-at="2016-02-04T16:03:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestCaseB -->
        <test-method status="PASS" signature="fourthTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="fourthTestCaseB" duration-ms="2002" started-at="2016-02-04T16:03:39Z" finished-at="2016-02-04T16:03:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fourthTestCaseB -->
        <test-method status="FAIL" signature="secondTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="secondTestCaseB" duration-ms="506" started-at="2016-02-04T16:03:41Z" finished-at="2016-02-04T16:03:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failing this Test]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failing this Test
	at org.testng.Assert.fail(Assert.java:94)
	at com.bility.test.sanityTests.TestB.secondTestCaseB(TestB.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- secondTestCaseB -->
        <test-method status="PASS" signature="thirdTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestB@56f4468b]" name="thirdTestCaseB" duration-ms="2003" started-at="2016-02-04T16:03:41Z" finished-at="2016-02-04T16:03:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thirdTestCaseB -->
      </class> <!-- com.bility.test.sanityTests.TestB -->
      <class name="com.bility.test.sanityTests.TestC">
        <test-method status="PASS" signature="firstTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestC@6cc4c815]" name="firstTestCaseB" duration-ms="102" started-at="2016-02-04T16:03:43Z" finished-at="2016-02-04T16:03:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestCaseB -->
        <test-method status="SKIP" signature="secondTestCaseB()[pri:0, instance:com.bility.test.sanityTests.TestC@6cc4c815]" name="secondTestCaseB" duration-ms="5004" started-at="2016-02-04T16:03:43Z" finished-at="2016-02-04T16:03:48Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping this test with exception]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping this test with exception
	at com.bility.test.sanityTests.TestC.secondTestCaseB(TestC.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- secondTestCaseB -->
      </class> <!-- com.bility.test.sanityTests.TestC -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
